bootstrap-salt.sh

```diff
@@ -2476,24 +2450,12 @@ install_ubuntu_stable_deps() {
 install_ubuntu_daily_deps() {
     install_ubuntu_stable_deps || return 1
 
-    if [ "$DISTRO_MAJOR_VERSION" -ge 12 ]; then
-        # Above Ubuntu 11.10 add-apt-repository is in a different package
-        __apt_get_install_noinput software-properties-common || return 1
-    else
-        __apt_get_install_noinput python-software-properties || return 1
-    fi
```

Comment:
This code here is for a reason, just the expression and comment are little bit wrong.
On Ubuntu 12.04 you need to install `python-software-properties` to get `add-apt-repository` binary:
http://packages.ubuntu.com/precise/all/python-software-properties/filelist

It is not in the `software-properties-common` pkg though: http://packages.ubuntu.com/precise/all/software-properties-common/filelist

The `if` statement should be:
```sh
if [ "$DISTRO_MAJOR_VERSION" -gt 12 ]; then
    ...
```